### PR TITLE
Feature/input focus

### PR DESCRIPTION
**Pull request by jamiesmyln**
Refactored and improved inputs

### CODE REVIEW

File: packages/react-magma-dom/src/components/TimePicker/index.tsx [hunk 1]
```diff
@@ -326,13 +316,14 @@ export const TimePicker = React.forwardRef<HTMLInputElement, TimePickerProps>(
...
             inputStyle={{
               background: `${theme.colors.neutral08}`,
-              borderColor: `${
-                errorMessage
-                  ? theme.colors.danger
-                  : isInverse
-                  ? theme.colors.neutral08
-                  : theme.colors.neutral04
-              }`,
+              // border: '1px solid',
```
Comment (joasmi723) [inline packages/react-magma-dom/src/components/TimePicker/index.tsx:319]: Can we remove these commented lines?  Other than that, this looks good to me.

Comment (jamiesmyln) [reply packages/react-magma-dom/src/components/TimePicker/index.tsx:319]: Sure thing

**Review by joasmi723 — APPROVED** [review]

**Comment by orion-cengage** [comment]
Can we please make the following adjustments to the SHOW button in order to optimize how the space within the input is being used?

Basically I think changing the button to the small variant will help the most, and maybe some small tweaks to spacing.  Below is an illustration of where the component is now, and where I'd like it to get.

Password Input - Button update

**Review by orion-cengage — CHANGES_REQUESTED** [review]
I requested some changes in the comments.

**Pull request merged**